src/CitMovie.Api/Controller/BookmarkController.cs

```diff
@@ -108,8 +108,8 @@ public async Task<IActionResult> DeleteBookmark(int userId, int id)
         if (bookmark.UserId != userId)
             return Forbid();
 
-        var deleted = await _bookmarkManager.DeleteBookmarkAsync(id);
-        return deleted ? NoContent() : NotFound();
```

Comment:
Why was this changed?

Reply:
Beacuse if everything is succesfull i just wanna return an ok with the below string.

Reply:
But what if it wasn't, cant really see there has been added any logic to account for bookmark failing to update. Also the correct status code is `NoContent` not `Ok` since its an api and not a ui that needs to display a html page or something like that

Reply:
There is still logic for if it fails. 

```Csharp
if (userId != GetUserId())
            return Forbid();
            
var bookmark = await _bookmarkManager.GetBookmarkAsync(id);

if (bookmark == null) 
            return NotFound();
            
if (bookmark.UserId != userId)
            return Forbid();
```

Reply:
Changed it to respond with 204 instead of 200.